src/vivarium_gates_mncnh/components/mortality.py

```diff
@@ -249,6 +257,17 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
             },
             index=pop_data.index,
         )
+        pregnancy_outcomes = self.population_view.subview([COLUMNS.PREGNANCY_OUTCOME]).get(
```

Comment:
This fixes a small record keeping issue where neonates that were partial or stillbirth would be labeled as dead but their cause of death would be not dead.